inference/worker/settings.py

```diff
@@ -17,5 +17,8 @@ class Settings(pydantic.BaseSettings):
     perform_oom_test: bool = False
     oom_test_max_length: int | None = None
 
+    # for hf basic server
+    quantize: bool = False
```

Comment:
Should we have a more descriptive setting name here so people don't expect it to have an effect when not using the basic server?

Reply:
this needs to be called quantize because the hf-inference server also expects it to be called like this